src/DDserver.cpp

```diff
@@ -262,13 +261,15 @@ namespace DoubleD
                     {
                         timeout = boost::lexical_cast<double>(req.url_params.get("timeout"));
                     }
-
                     else
                     {
                         timeout = 0.0f;
                     }
-
-                    x[DDserver::m_server_name][0] = DDserver::m_handleRequest(timeout, lockName, lifetime);
+                    
+                    boost::optional<Lock> lock = DDserver::m_handleRequest(lockName, timeout, lifetime);
+                    x["sessiontoken"] = lock ? lock.get().m_getSessionToken() : "";
```

Comment:
I actually would prefer to return json `null`, when there is no lock/sessionToken but I can't figure out how to. Hence the empty string  ;)